topics/community/tutorials/community_content/tutorial.md

```diff
@@ -14,10 +14,10 @@ time_estimation: "30M"
 key_points:
```

Comment:
this file shouldn't be included here right?

Reply:
100%